modules/virt-creating-interface-on-nodes.adoc

```diff
@@ -6,11 +6,16 @@
 [id="virt-creating-interface-on-nodes_{context}"]
 = Creating an interface on nodes
 
-Create an interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` manifest to the cluster. The manifest details the requested configuration for the interface.
+Create an interface on nodes in the cluster by applying a `NodeNetworkConfigurationPolicy` (NNCP) manifest to the cluster. The manifest details the requested configuration for the interface.
 
 By default, the manifest applies to all nodes in the cluster. To add the interface to specific nodes, add the `spec: nodeSelector` parameter and the appropriate `<key>:<value>` for your node selector.
 
-You can configure multiple nmstate-enabled nodes concurrently. The configuration applies to 50% of the nodes in parallel. This strategy prevents the entire cluster from being unavailable if the network connection fails. To apply the policy configuration in parallel to a specific portion of the cluster, use the `maxUnavailable` field.
+You can configure multiple nmstate-enabled nodes concurrently. The configuration applies to 50% of the nodes in parallel. This strategy prevents the entire cluster from being unavailable if the network connection fails. To apply the policy configuration in parallel to a specific portion of the cluster, use the `maxUnavailable` parameter in the `NodeNetworkConfigurationPolicy` manifest configuration file.
+
+[NOTE]
+====
+If you set the `maxUnavailable` parameter to `50%` in two NNCP manifests that each apply to the same node, the policy configuration coverage is 100% for that node. To avoid this situation, ensure that only one NNCP manifest applies to a node. 
```

Comment:
I'm still not sure this example is clear. I think the problem is that a single node can't really represent what's happening. This issue is that if you have two nodes, and an NNCP named A that applies to both of them, with maxUnavailable set to 50%, then at most only one of the nodes will ever have the config applied to it at one time.

However, if you then add another NNCP B with maxUnavailable set to 50%, it's possible both nodes will have config applied at the same time. One node might get A, one node might get B, and because they're both 50% that's completely valid (and also probably unexpected).